packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -76,6 +76,12 @@ export interface ParticipantItemStrings {
   /* @conditional-compile-remove(PSTN-calls) */
   /** String shown when `participantState` is `Hold` */
   participantStateHold?: string;
+  /* @conditional-compile-remove(raise-hand) */
+  /** Label for the remove button in participant menu  */
+  lowerParticipantHandButtonLabel: string;
```

Comment:
I see interface changes in `ParticipantItem.tsx` but no functional change in `ParticipantItem.tsx`. If someone is just using the participantitem component and not the participantlist componet, how would they use these? Some of the logic in ParticipantList may need to be moved to ParticipantItem (or vice-versa, these strings in participantItem need moved to participantsList)